active/003-seq-feature.md

```diff
@@ -0,0 +1,108 @@
+---
+Feature name: interval-metadata
+Start date: <2015-12-18>
+Pull request: 1
+Authors: ["@RNAer", "@mortonjt"]
+Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
+---
+
+# Summary
+
+Introduce an `interval_metadata` object to allow for the storage and querying of intervals.
```

Comment:
> I am fine with the naming, but better naming is always welcome : D

Which naming are you fine with? We were originally disagreeing on what name to use.

I want to figure this out because this name should match the mental model we're creating for users. Does a user inherently understand this type of metadata as 1) a collection of intervals on a sequence that exist in one or more features, or 2) a collection of features on a sequence, where each feature exists over some set of intervals? (I'm pushing for the latter).